README.md

```diff
@@ -43,4 +43,30 @@ mvn spotless:apply
 
 ## License
 
-[BSD 3-clause](/LICENSE.md). The organization of layers and features used by these map styles, as well as the "look and feel" of the resulting maps, are licensed [CC0](https://creativecommons.org/publicdomain/zero/1.0/). However, maps using the [Protomaps web map service](https://protomaps.com) or another OpenStreetMap-based service will be subject to the terms of the [Open Database License](https://www.openstreetmap.org/copyright).
+# Licensing and Attribution Guidelines
+
+Summary:
+
+* All code is BSD-3.
+* Map design is CC0.
+* Tilesets are ODbL, attribute OSM.
+* We like shoutouts!
+* Name your fork, commercial product or service something different.
+
+We kindly request that you attribute the Protomaps project if you use the basemap styles unmodified or with additions, but you are not required to do so. The visual design copyright of the maps is released under a [Creative Commons Zero (CC0) license](https://creativecommons.org/publicdomain/zero/1.0/) to eliminate ambiguity around derivative or combined works of map styling code, and to publish software, such as NPM packages, under the standard BSD-3 license.
+
+The tilesets that power the Protomaps basemap are [Produced Works of the OpenStreetMap dataset](https://osmfoundation.org/wiki/Licence/Community_Guidelines/Produced_Work_-_Guideline) under the [Open Database License](https://www.openstreetmap.org/copyright). Web maps and native apps that use this Produced Work must visibly attribute © OpenStreetMap - for example, in the corner of the map display. If your map only uses the Map Styles with non-OpenStreetMap tilesets, this attribution is not required.
```

Comment:
Since the basemap does use `Natural Earth` already, then we should at least mention it here and link to it. 

It's OK to not include it in the example web map corner attribution, though.